core/plugins/org.polarsys.capella.core.validation.ui/src/org/polarsys/capella/core/validation/ui/DynamicActionProvider.java

```diff
@@ -114,10 +115,11 @@ public ValidateAction createDefaultValidation() {
    * @return
    */
   protected IContributionItem createContributionItem() {
-
     if (userValidationActions.size() > 0) {
       IMenuManager menuManager = new MenuManager("Validate Model", imageDescriptor, "ID"); //$NON-NLS-1$ //$NON-NLS-2$
-      menuManager.add(defaultValidationAction);
+      if (defaultValidationAction.isEnabled()) {

```

Comment:
What is the reason behind this change?

Reply:
The action si grayed out if disable, a menu si created but does nothing.